WordPress/openverse-frontend
# Add a Tabs component

## Fixes


Fixes #749 by @sarayourfriend 
Fixes #1000 by @zackkrida 

## Description


This PR adds the accessible Tabs component and uses it for **_License attribution copy_** component and the **_Feedback_** page. This implementation is based on the Tailwind Labs [Headless UI ](https://headlessui.dev/) Tabs component implementation (MIT licensed).
The `focus-management` file added probably duplicates a lot of functionality that we are already using from AriaKit.

## Testing Instructions

The number of changed files might seem intimidating, but it's mostly snapshot and tape files.

Open a single result page (audio and image) and check that the Copy attribution tabs work correctly, and you can move through the tabs using arrow keys. Arrow keys move in the correct direction in RTL pages, too! When you <kbd>Tab</kbd> from a selected tab, you focus the tabpanel.

Open the feedback page and see that the tab panels are working correctly. To open a tab using the keyboard you need to first use your arrow keys to focus on the tab, and then click `Enter` or `Space` to actually select it.

Run the Storybook and check that the two tab stories work correctly.

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
